azure-spring-boot/pom.xml

```diff
@@ -142,6 +142,31 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>applicationinsights-core</artifactId>
+            <version>2.2.0-SNAPSHOT</version>
```

Comment:
Trying to test with latest build of AI. Will remove this in polish comment, once other important issues are solved.

Reply:
I have just removed this and the CI/CD now passes.